test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterYourContactDetailsControllerSpec.scala

```diff
@@ -50,13 +55,15 @@ class EnterClaimantDetailsAsImporterCompanyControllerSpec
       bind[SessionCache].toInstance(mockSessionCache)
     )
 
-  lazy val controller: EnterClaimantDetailsAsImporterCompanyController =
-    instanceOf[EnterClaimantDetailsAsImporterCompanyController]
+  lazy val controller: EnterYourContactDetailsController =
+    instanceOf[EnterYourContactDetailsController]
 
   implicit lazy val messagesApi: MessagesApi = controller.messagesApi
 
   implicit lazy val messages: Messages = MessagesImpl(Lang("en"), messagesApi)
 
+  val signedInUserEmail = "he.who.shall@not.be.named"
```

Comment:
I believe there is a generator for this so you don't need to make up a value yourself